kernel/tests/golden_tables.rs

```diff
@@ -316,9 +327,11 @@ golden_test!("data-reader-date-types-utc", latest_snapshot_test);
 golden_test!("data-reader-escaped-chars", latest_snapshot_test);
 skip_test!("data-reader-map": "map field named 'entries' vs 'key_value'");
```

Comment:
if you want I think we can turn this on and it should just work (needed list and map fixes)

Reply:
```suggestion
golden_test!("data-reader-map", latest_snapshot_test);
```

Reply:
nice yep!